### PR TITLE
Migrated email to Office365

### LINKED ISSUE #65: Migrate email system to office365

### PR DESCRIPTION
**Pull request by arav-sharma**

### CODE REVIEW

File: helperFunctions.js [hunk 1]
```diff
@@ -2,20 +2,20 @@ const nodemailer = require('nodemailer'); //Sending Email Conformations and car
...
-    service: 'gmail',
+    host: 'smtp.office365.com',
+    port: 587,
+    secure: false, // Set to true if you want to use TLS (not recommended for port 587)
     auth: {
-      user: 'ronshaked07@gmail.com',
-      pass: 'sknf irxh yhaa uhjq'
-    }
+      user: 'donotreply@automatch.dev', // Your custom email address
+      pass: 'e8$6rRgQVd#4tF2', // Your Microsoft 365 email password
```
Comment (RonShk) [inline helperFunctions.js]: Is this a password for the account itself, if so find a different way to link the email if not and its like the gmail App Passcodes than thats fine, o

File: helperFunctions.js [hunk 2]
```diff
@@ -2,20 +2,20 @@ const nodemailer = require('nodemailer'); //Sending Email Conformations and car
...
+  
   const details = {
-    from: 'ronshaked07@gmail.com',
+    from: 'donotreply@automatch.dev', // Your custom email address
     to: userEmail,
-    subject: 'Confirmation Email',
-    text: `Click the link below to confirm your account:\n${confirmationLink}`,
-    html: `Click the link below to confirm your account:<br><a href="${confirmationLink}">Confirm your account</a>`
+    subject: 'AutoMatch - Verify your Account',
+    text: `Click here to verify your account: http://localhost:8080/confirm/user?objectid=${objectID}`
```
Comment (RonShk) [inline helperFunctions.js]: Make it so it says "Confirm Account" instead of a sketchy url, (Feedback I got from CS Forum people)

**Comment by RonShk** [comment]
Another thing I forgot to mention, what is port 587? Why specifacally that port?

File: helperFunctions.js (hunk 2 above)
Comment (arav-sharma) [reply helperFunctions.js]: Done

**Comment by arav-sharma** [comment]
> Another thing I forgot to mention, what is port 587? Why specifacally that port?

its just commonly used for invoking emails in office365, its nothing important

**Pull request merged**